scripts/pre_commit_linter.py

```diff
@@ -180,6 +182,16 @@
 _MESSAGE_TYPE_FAILED = 'FAILED'
 
 
+def _is_filename_excluded_for_bad_patterns_check(filename):
+    """Checks if file is excluded from the bad patterns check
+
+    Returns:
+        a boolean: true if excluded
+    """
+    return (any(filename.startswith(bad_pattern)
+                for bad_pattern in BAD_PATTERNS['\t']['excluded_dirs'])
```

Comment:
Just to double check: underneath `def main()`, if you put `print len(BAD_PATTERNS['\t']['excluded_dirs'])`, you get 1? I get 12.